docs/getting-started-guides/kubeadm.md

```diff
@@ -8,12 +8,24 @@ li>.highlighter-rouge {position:relative; top:3px;}
 ## Overview
 
 This quickstart shows you how to easily install a secure Kubernetes cluster on machines running Ubuntu 16.04 or CentOS 7.
-The installation uses a tool called `kubeadm` which is part of Kubernetes 1.4.
+The installation uses a tool called `kubeadm` which is alpha as of Kubernetes 1.4.
```

Comment:
Yeah, I was trying to side-step the issue of what does it mean for something to be "part of" kubernetes 1.4 vs not.  I'll revert as this is a huge hairball...